pages/episodes/[slug].js

```diff
@@ -22,7 +22,7 @@ export default function EpisodeDetailsPage() {
     // this logic will change when using remote API
     const [filteredEpisode] = serie
```

Comment:
filteredEpisode can be undefined, since slug can be undefined (This happens when you reload the episode details page directly via the browser). This leads to an error in the EpisodeDetails component. You should either make sure that the slug is not empty or use conditional rendering for the usage of the EpisodeDetails component.

Reply:
fixed it with `if(router.isReady) ...` hope this will do